src/test/java/uk/gov/laa/ccms/caab/builders/ApplicationSummaryBuilderTest.java

```diff
@@ -0,0 +1,143 @@
+package uk.gov.laa.ccms.caab.builders;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.sql.Date;
+import java.time.Instant;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import uk.gov.laa.ccms.caab.model.Address;
+import uk.gov.laa.ccms.caab.model.ApplicationSummaryDisplay;
+import uk.gov.laa.ccms.caab.model.AuditDetail;
+import uk.gov.laa.ccms.caab.model.CostStructure;
+import uk.gov.laa.ccms.caab.model.Opponent;
+import uk.gov.laa.ccms.caab.model.PriorAuthority;
+import uk.gov.laa.ccms.caab.model.Proceeding;
+import uk.gov.laa.ccms.caab.model.StringDisplayValue;
+import uk.gov.laa.ccms.data.model.RelationshipToCaseLookupValueDetail;
+
+class ApplicationSummaryBuilderTest {
+
+  private ApplicationSummaryBuilder builder;
+
+  @BeforeEach
+  void setUp() {
+    // Mock an AuditDetail for testing
+    AuditDetail auditDetail = new AuditDetail();
+    auditDetail.setLastSaved(Date.from(Instant.now()));
+    auditDetail.setLastSavedBy("TestUser");
+
+    // Create a new builder for each test
+    builder = new ApplicationSummaryBuilder(auditDetail);
+  }
+
+  @Test
+  void testClientFullName() {
```

Comment:
a bit pedantic, but going back to my earlier comment, can either of the name values be null and will the builder handle it?

Reply:
I have amended the builder now to handle the null strings